<?xml version="1.0"?>
<launch>
    <arg name="nodelet_manager" default="velodyne_nodelet_manager"/>

	<node pkg="nodelet" type="nodelet" name="octomap_server_nodelet" 
    args="load octomap_server/OctomapServerNodelet $(arg nodelet_manager)">
		<param name="resolution" value="0.2" />
		<param name="frame_id" type="string" value="odom" />
		<param name="sensor_model/max_range" value="20.0" />
		<!-- <param name="sensor_model/min" value="0.1" /-->
		<!-- param name="sensor_model/max" value="0.4" /-->
		<param name="sensor_model/hit" value="0.7" />
		<param name="sensor_model/miss" value="0.2" /> -->
		<remap from="cloud_in" to="/filtered_points" />
	</node>

</launch>